src/javascript/app_2/Stores/Modules/SmartChart/Helpers/__tests__/barriers.js

```diff
@@ -0,0 +1,29 @@
+import { expect }                               from 'chai';
+import React                                    from 'react';
+import { isBarrierSupported, barriersToString } from '../barriers';
+
+describe('Barriers', () => {
+    describe('isBarrierSupported', () => {
+        it('should return false when barrier is not in CONTRACT_SHADES', () => {
+            expect(isBarrierSupported('SomeThinMadeUp')).to.eql(false);
+        });
+        it('should return true when barrier is in CONTRACT_SHADES', () => {
+            expect(isBarrierSupported('CALL')).to.eql(true);
+        });
+    });
+
+    describe('barriersToString', () => {
+        it('should convert non-zero barriers which do not have +/- to string consisting of them without +/- while is_relative is false', () => {
+            expect(barriersToString(false, [10, 15])).to.deep.eql(['10','15']);
```

Comment:
@nazaninreihani Seems `barriersToString()` doesn't expect to receive an array as the second argument, since it is a rest parameter:
https://github.com/binary-com/binary-static/blob/a0437f82d150fc44135659fcc54b9b68cc61f236/src/javascript/app_2/Stores/Modules/SmartChart/Helpers/barriers.js#L7

So, calling it like this: `barriersToString(false, 15, 10)` should work correctly.

Another sample call:
https://github.com/binary-com/binary-static/blob/a0437f82d150fc44135659fcc54b9b68cc61f236/src/javascript/app_2/Stores/Modules/SmartChart/chart_barrier_store.js#L67